Getting package deprecated warning

## Alamofire Environment

**Alamofire Version:5.10.1**
**Dependency Manager:**
**Xcode Version:Version 16.0 (16A242d)**
**Swift Version:5.10**
**Platform(s) Running Alamofire:5.10.1**
**macOS Version Running Xcode:Mac-mini Apple M2 Pro with MacOS 15.0.1 (24A348)**

## Terminal Warning while swift build

```sh
mehuls-mac-mini:mobile-sdk-ios mehul$ swift build
warning: 'alamofire': /Users/mehul/Documents/iOS/Projects/LLC/mobile-sdk-ios/.build/checkouts/Alamofire/Package.swift:28:15: warning: 'init(name:defaultLocalization:platforms:pkgConfig:providers:products:dependencies:targets:swiftLanguageVersions:cLanguageStandard:cxxLanguageStandard:)' is deprecated: replaced by 'init(name:defaultLocalization:platforms:pkgConfig:providers:products:dependencies:targets:swiftLanguageModes:cLanguageStandard:cxxLanguageStandard:)'
26 | import PackageDescription
27 | 
28 | let package = Package(name: "Alamofire",
   |               |- warning: 'init(name:defaultLocalization:platforms:pkgConfig:providers:products:dependencies:targets:swiftLanguageVersions:cLanguageStandard:cxxLanguageStandard:)' is deprecated: replaced by 'init(name:defaultLocalization:platforms:pkgConfig:providers:products:dependencies:targets:swiftLanguageModes:cLanguageStandard:cxxLanguageStandard:)'
   |               `- note: use 'init(name:defaultLocalization:platforms:pkgConfig:providers:products:dependencies:targets:swiftLanguageModes:cLanguageStandard:cxxLanguageStandard:)' instead
29 |                       platforms: [.macOS(.v10_13),
30 |                                   .iOS(.v12),
warning: 'objectmapper': found 2 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
    /Users/mehul/Documents/iOS/Projects/LLC/mobile-sdk-ios/.build/checkouts/ObjectMapper/Sources/Info.plist
    /Users/mehul/Documents/iOS/Projects/LLC/mobile-sdk-ios/.build/checkouts/ObjectMapper/Sources/Resources/PrivacyInfo.xcprivacy
Building for debugging...
[1/1] Write swift-version--58304C5D6DBC2206.txt
Build complete! (2.48s)

```

![Screenshot 2024-10-22 at 6 51 05 PM](https://github.com/user-attachments/assets/13561438-c874-4875-acfd-0115075bdb6d)

--- a/Package.swift
+++ b/Package.swift
@@ -49,4 +49,4 @@ let package = Package(name: "Alamofire",
                                             path: "Tests",
                                             exclude: ["Info.plist", "Test Plans"],
                                             resources: [.process("Resources")])],
-                      swiftLanguageVersions: [.v5])
+                      swiftLanguageModes: [.v5])

